README.md

```diff
@@ -2,31 +2,24 @@
 
 This repository contains the Softward Development Kit to interact with the Aragon protocol.
 
-It currently provides support for JavaScript and TypeScript. [Learn more](./javascript). 
+It currently provides support for JavaScript and TypeScript. [Learn more](./javascript).
 
-## Pull Request Labels
+## Automatic publishing
 
-To handle certain automated tasks, you can add labels to your pull requests. Here is an overview of the available labels and their meaning.
+### With labels
 
-### JavaScript tags
+To automtically publish the new version on pull request merge add one or all of these labels:
+| Label Name | Package that gets published |
+| --- | --- |
+| client-publish | `javascript/modules/client/` |
+| common-publish | `javascript/modules/common/` |
 
-| Label               | Meaning                                                          | Remarks                                          |
-| ------------------- | ---------------------------------------------------------------- | ------------------------------------------------ |
-| `<component>:patch` | Triggers a new NPM patch version of the corresponding JS package | Don't add any other `<component>:*` label |
-| `<component>:minor` | Triggers a new NPM minor version of the corresponding JS package | Don't add any other `<component>:*` label |
-| `<component>:major` | Triggers a new NPM major version of the corresponding JS package | Don't add any other `<component>:*` label |
+### With tags
 
-The available components are:
+To publish a new version of a subpackage create a new git tag following this schema:  
+`VERSION-LANGUAGE-PACKAGE_FOLDER_NAME`  
+Example to publish a new version of the javascript client module: `0.0.1-javascript-client`.  
+To publish an alpha version just put `-alpha` behind the semver: `0.0.1-alpha-javascript-client`
```

Comment:
Is this being implemented by the updated codebase?

Reply:
this was already there but not documented